provision/ec2_test.go

```diff
@@ -15,3 +15,7 @@ func TestNewEC2ProvisionerFromEnv(t *testing.T) {
 	prov, _ := NewEC2Provisioner("id", "key")
 	assert.NotNil(t, prov.client.Config.Credentials)
 }
+
+func TestNewEC2ProvisionerFromProfile(t *testing.T) {
+	_, _ = NewEC2ProvisionerFromProfile("", "")
```

Comment:
Great test

Reply:
Thanks - I think testing this properly is a bit of a stretch goal for now, since the API is a pain in the ass